lang/cs/Org.Apache.REEF.Tang.Tests/Configuration/TestConfiguration.cs

```diff
@@ -425,6 +426,77 @@ public void TestTimerConfigurationWithClassHierarchy()
             timer.sleep();
         }
 
+        [Fact]
+        public void TestListConfig()
+        {
+            IList<string> stringList1 = new List<string>();
+            stringList1.Add("foo");
+            stringList1.Add("bar");
+
+            IList<string> stringList2 = new List<string>();
+            stringList2.Add("test1");
+            stringList2.Add("test2");
+            stringList2.Add("test3");
+            stringList2.Add("test4");
+
+            IConfiguration conf = TangFactory.GetTang().NewConfigurationBuilder()
+                .BindList<ListOfStrings, string>(GenericType<ListOfStrings>.Class, stringList1)
+                .BindList<ListOfStrings2, string>(GenericType<ListOfStrings2>.Class, stringList2)
+                .Build();
+
+            ConfigurationFile.WriteConfigurationFile(conf, "ListOfStringsConf.txt");
```

Comment:
This file should be deleted by the end of the tests. Or, better yet, serialize the configuration to a `byte[]` and back.

Reply:
This is not needed -- I actually convert the config to a string and then read back in the string. So I have deleted this line.

---
In reply to: [223778711](https://github.com/apache/reef/pull/1479#discussion_r223778711) [](ancestors = 223778711)